resources/qml/TertiaryButton.qml

```diff
@@ -9,13 +9,11 @@ import Cura 1.1 as Cura
 
 Cura.ActionButton
 {
-    shadowEnabled: true
-    shadowColor: enabled ? UM.Theme.getColor("secondary_button_shadow"): UM.Theme.getColor("action_button_disabled_shadow")
     color: "transparent"
     textColor: UM.Theme.getColor("text_link")
     outlineColor: "transparent"
-    disabledColor: UM.Theme.getColor("action_button_disabled")
+    outlineDisabledColor: "transparent"
+    disabledColor: "transparent"
     textDisabledColor: UM.Theme.getColor("action_button_disabled_text")
-    hoverColor: "transparent"
-    underlineTextOnHover: true
```

Comment:
These links are underlined in the design. Should they now not always get underlined (regardless of hover)?

Reply:
From the ticket:
> we do not have tertiary buttons in cura (yet).

This is a bit of a lie. We _do_ have (some) tertiary buttons, and they _are_ used differently than links/quaternary. For example, the 'Skip' button in the what's new pages is tertiary, but should not be a link:

> These links ...

... and they aren't really links either. (As evidenced by the fact that the 'old' tertiary button _also_ didn't have underlining in any 'mode' anyway.)

In any case, everything that was a link will stay a link, because _those_ where never using (this) tertiary in the first place. 

So: I just took the ticket and comments to mean that I shouldn't yet add Tertiary buttons in places where we now use links that we maybe _should_ use them.

Reply:
Sorry, that part about the old button was wrong. I think the rest still stands though?

Reply:
Sorry, that part about the old button was wrong. I think the rest still stands though?

Reply:
Sorry, that part about the old button was wrong. I think the rest still stands though?

Reply:
I do agree with Ruben Dulek here that the tertiary button should remain the "link" button, as it was traditionally used, and we should introduce the quatertiary for the more subtle action button. If we change the Tertiary buttons to not be links, the existing tertiary buttons no longer look like they link to an external source:

![image](https://user-images.githubusercontent.com/19388042/124574652-fa1bbc80-de4a-11eb-8dc9-e2a1d9de9bac.png)

This is probably not desirable by plugins that use Tertiary buttons.

Reply:
Oof, sorry about all of that. I was completely in the wrong here :-(